framework/src/meshmodifiers/SideSetsAroundSubdomain.C

```diff
@@ -58,6 +62,9 @@ SideSetsAroundSubdomain::modify()
   if (!_mesh_ptr)
     mooseError("_mesh_ptr must be initialized before calling SideSetsAroundSubdomain::modify()!");
 
+  // Initialize the BlockRestrictable parent
+  initializeBlockRestrictable(_pars);
```

Comment:
I mean - you're already doing something similar with `forcePrepare()`... that could have been done inside the MeshModifier too...

I don't really see what you mean... we call `initialize()` and all kinds of other "initialization" routines on objects before asking them to compute all the time.

It's not over-designing... this design isn't right.  `modify()` should not be doing "initialization" stuff.

I will agree that it's a minor issue though (no design is perfect)...